### ISSUE TITLE
Unable to copy and paste HTML in IE with draftjs

### ISSUE THREAD

**Issue by VassilIordanov** [NCI-Agency/anet#1114]
Reproducible in IE11, maybe in others. True for any kind of copy/paste, even internally in drafts

**Comment by evdhoudt** [comment]
Cannot reproduce the error on Osx. Not even when using Browserstack and IE11. I suspect that this is an operating issue with Windows clipboard.

**Comment by dspCarlson** [comment]
Yesterday (4 April 2019) a user notified me that he had lost all the text in the "Engagement details" data field when he pasted from Microsoft Word.

Today, I was able to recreate the bug in the biography, engagement details and sensitive information data fields.

I was not able to recreate the bug in the meeting goal, key outcomes and next steps data fields.

As an example to recreate the bug, type some words in the "Engagement details" data field.  At the end or below the words just entered, paste words copied from Word.  The words pasted from Word will be displayed but the words already entered will no longer be displayed.

I am closing the original issue I created, i.e. #1567
